config/api.yml

```diff
@@ -1880,6 +1880,30 @@
         :identifier: host_aggregate_tag
       - :name: unassign
         :identifier: host_aggregate_tag
+  :host_initiators:
+    :description: Host Initiators
+    :identifier: host_initiator
+    :options:
+    - :collection
+    :verbs: *gp
+    :klass: HostInitiator
+    :subcollections:
+    :collection_actions:
+      :get:
+      - :name: read
+        :identifier: host_initiator_show_list
+      :post:
+      - :name: create
+        :identifier: host_initiator_new
+      - :name: refresh
+        :identifier: host_initiator_refresh
```

Comment:
What does a refresh action on the collection mean?  I would expect refresh only on the resource

Reply:
Ahh okay I see you're doing a bulk refresh action https://github.com/ManageIQ/manageiq-api/pull/985/files#diff-3dbf099b30332a2d5c69d3f876bb4510c784579cd46a1c9f5ae6bddb399be467R102
Carry on